### DISCUSSION TITLE
Multiple source repo, revision not visible

### DISCUSSION THREAD

**Discussion by Shubham1406220** [argoproj/argo-cd#13177] [Q&A]
Hello, 

I have setup the multiple source repo setup in applicationset. When notification is triggered the revision section is blank but it is visible for single repo in applicationset. Have i missed any configuration to be setup or it is the limitation of multi source repo setup.

I also want to  show the image tag of the application got deployed  in notification for the same setup. Is it possible ??

**Comment by jaideepr97** [comment]
Hi @Shubham1406220 thanks for raising this
Sounds like this could be something that was missed during the multi source implementatiion
I would suggest creating an issue for it

**Reply by Shubham1406220** [reply]
sure @jaideepr97
